src/modules/roc_pipeline/config.h

```diff
@@ -90,6 +91,12 @@ struct SenderConfig {
     //! Fill unitialized data with large values to make them more noticable.
     bool poisoning;
 
+    //! Profile moving average of frames being written.
+    bool profiling;
+
+    //! Profiler configuration.
+    struct audio::ProfilerConfig profiler_config;
```

Comment:
Here and in other places, `struct` keyword is not needed. BTW, `profiler_config()` initializer is redundant too.